```
AILED: /usr/lib/ccache/clang -DINCLUDE_GENERATED_DECLARATIONS -DMIN_LOG_LEVEL=3 -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -D_GNU_SOURCE -Iconfig -I../src -Isrc/nvim/auto -Iinclude -isystem /home/travis/nvim-deps/usr/include -g   -DEXITFREE -fno-sanitize-recover=all -fno-omit-frame-pointer -fno-optimize-sibling-calls -fsanitize=address -fsanitize=undefined -fsanitize-blacklist=/home/travis/build/neovim/neovim/src/.asan-blacklist -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fdiagnostics-color=always -Werror  -Wno-conversion -MD -MT src/nvim/CMakeFiles/nvim.dir/diff.c.o -MF src/nvim/CMakeFiles/nvim.dir/diff.c.o.d -o src/nvim/CMakeFiles/nvim.dir/diff.c.o   -c ../src/nvim/diff.c
../src/nvim/diff.c:124:32: error: self-comparison always evaluates to true [-Werror,-Wtautological-compare]
    for (win_T *wp = ((curtab) == curtab) ? firstwin : (curtab)->tp_firstwin; wp != ((void*)0); wp = wp->w_next) {
                               ^
../src/nvim/diff.c:638:30: error: self-comparison always evaluates to true [-Werror,-Wtautological-compare]
  for (win_T *wp = ((curtab) == curtab) ? firstwin : (curtab)->tp_firstwin; wp != ((void*)0); wp = wp->w_next) {
                             ^
../src/nvim/diff.c:1418:30: error: self-comparison always evaluates to true [-Werror,-Wtautological-compare]
  for (win_T *wp = ((curtab) == curtab) ? firstwin : (curtab)->tp_firstwin; wp != ((void*)0); wp = wp->w_next) {
                             ^
../src/nvim/diff.c:2769:30: error: self-comparison always evaluates to true [-Werror,-Wtautological-compare]
  for (win_T *wp = ((curtab) == curtab) ? firstwin : (curtab)->tp_firstwin; wp != ((void*)0); wp = wp->w_next) {
                             ^
4 errors generated.
[143/262] Building C object src/nvim/CMakeFiles/nvim.dir/cursor_shape.c.o
ninja: build stopped: subcommand failed.
+exit 1
```

Similar to #10480 I cannot reproduce this locally (clang version 8.0.0 (tags/RELEASE_800/final)).

In both cases it seems like locally this is ignored with macros.

--- a/.travis.yml
+++ b/.travis.yml
@@ -34,7 +34,6 @@ env:
     - ASAN_OPTIONS="detect_leaks=1:check_initialization_order=1:log_path=$LOG_DIR/asan"
     - TSAN_OPTIONS="log_path=$LOG_DIR/tsan"
     - UBSAN_OPTIONS="print_stacktrace=1 log_path=$LOG_DIR/ubsan"
-    - ASAN_SYMBOLIZE=asan_symbolize
     # Environment variables for Valgrind.
     - VALGRIND_LOG="$LOG_DIR/valgrind-%p.log"
     # If this file exists, the cache is valid (compile was successful).
@@ -62,7 +61,7 @@ addons:
       - autoconf
       - automake
       - build-essential
-      - clang-4.0
+      - clang
       - cmake
       - cscope
       - gcc-multilib
@@ -88,12 +87,11 @@ jobs:
     - stage: baseline
       name: clang-asan
       os: linux
-      compiler: clang-4.0
+      compiler: clang
       # Use Lua so that ASAN can test our embedded Lua support. 8fec4d53d0f6
       env:
         - CLANG_SANITIZER=ASAN_UBSAN
         - CMAKE_FLAGS="$CMAKE_FLAGS -DPREFER_LUA=ON"
-        - ASAN_SYMBOLIZE=asan_symbolize-4.0
         - *common-job-env
     - name: gcc-functionaltest-lua
       os: linux

--- a/ci/common/test.sh
+++ b/ci/common/test.sh
@@ -81,7 +81,7 @@ valgrind_check() {
 
 asan_check() {
   if test "${CLANG_SANITIZER}" = "ASAN_UBSAN" ; then
-    check_logs "${1}" "*san.*" | $ASAN_SYMBOLIZE
+    check_logs "${1}" "*san.*" | asan_symbolize
   fi
 }
 

